src/core/execution/SAMLauncherExecution.ts

```
@@ -271,35 +271,39 @@ export class SAMLauncherExecution implements Execution {
-        if (!isUnit(unit)) return false;
-        if (unit.owner() === this.player) return false;
-
-        // After game-over in team games, SAMs also target teammate MIRVs (aftergame fun)
-        const nukeOwner = unit.owner();
-        if (this.player.isFriendly(nukeOwner)) {
-          if (
-            this.mg.getWinner() === null ||
-            !this.player.isOnSameTeam(nukeOwner)
-          ) {
-            return false;
+    let mirvWarheadTargets = this.mg
```

ESLint

[error] 'mirvWarheadTargets' is never reassigned. Use 'const' instead.

(prefer-const)